.cargo/config

```diff
@@ -0,0 +1,2 @@
+[net]
+git-fetch-with-cli = true
```

Comment:
Why do you need that? For me, it compiles without it

Reply:
Good point, we can remove it, I needed it for when we were requiring an Arti with a dummy signature validation. (I kept this repo private as it is not intended to be used outside as for development.)